phoenicis-javafx/pom.xml

```diff
@@ -186,6 +186,16 @@
                 <jdk>10</jdk>
```

Comment:
It looks to me like the profiles for JDK 10 and JDK 11 are the same.
Can we unify them with `<jdk>10, 11</jdk>`?

Reply:
Probably yes but then we get another profile (additionally to jdk10 and jdk11). I think we will get rid of JDK10 quite soon anyways because it's just an intermediate step to 11 so not sure it's worth it.

Reply:
Then how about using `<jdk>[10,)</jdk>` for now?
We can later change this to `<jdk>[11,)</jdk>`, which not only contains JDK 11 but also all future JDKs.